src/common/transformations/src/transformations/transpose_sinking/ts_reduction.cpp

```diff
@@ -42,21 +42,21 @@ bool get_keep_dims(const std::shared_ptr<Node>& main_node) {
 TSReductionForward::TSReductionForward() {
     MATCHER_SCOPE(TSReductionForward);
 
-    auto transpose_label = wrap_type<Transpose>({any_input(), wrap_type<Constant>()});
+    auto transpose_label = wrap_type<ov::op::v1::Transpose>({any_input(), wrap_type<ov::op::v0::Constant>()});
```

Comment:
_A general comment which applies elsewhere_
Adding below piece of code at the beginning of file or function would avoid such changes in this file. Just an idea for further consideration.
```
using ov::op::v0::Constant;
using ov::op::v1::Transpose;
```